### ISSUE TITLE
Add support for JSX file format as well via create-email

### ISSUE THREAD

**Issue by gokulkrishh** [resendlabs/react-email#423]
Using this command, i am able to create starter email 

```bash
npx create-email@latest
```
By default, above command create Component files in `.tsx` file format, it would be great if there is flag to enable the support for `.jsx` file format as well

Suggestion:

```bash
npx create-email@latest --jsx
```

**Comment by bukinoshita** [comment]
Nice, thanks for creating this @gokulkrishh.

I think we can have a `typescript` flag.

```bash
$ create-email
```

```bash
$ create-email --typescript
```

**Comment by gokulkrishh** [comment]
@bukinoshita So by default `.JSX` and a flag for `typescript`. This would great.

**Comment by bastiaanv** [comment]
I would suggest to switch it around. Typescript by default, and `.JSX` as flag. Typescript is currently the default for most frameworks

**Comment by gokulkrishh** [comment]
@bastiaanv I am not so sure about typescript being the default for most frameworks (Not trying to start a debate on typescript 😄). 

For CRA and NextJS, we need to pass `--typescript`.

Would leave this decision to you people to decide.

**Comment by bukinoshita** [comment]
Yeah, I'd prefer having Typescript as default as well, but most libraries has JS as default instead of typescript. So I'd suggest going with `jsx` as default and in the example always show the `typescript` flag.
